project-base/storefront/components/Layout/Header/Navigation/Navigation.tsx

```diff
@@ -7,7 +7,7 @@ export type NavigationProps = {
 
 export const Navigation: FC<NavigationProps> = ({ navigation }) => {
     return (
-        <ul className="relative hidden w-full lg:flex lg:gap-6 xl:gap-12">
+        <ul className="relative hidden w-full lg:flex">
```

Comment:
It also means that the link can be now hovered even if the mouse cursor is actually far from the link. This is because the area between links is split between the two links (see video). I am not saying this is necessarily wrong, but I do not see this in ACs and I am not sure if we want it to behave this way.

https://github.com/user-attachments/assets/d3748d96-aac7-4288-85f9-ac73fa133897

Reply:
After consultation, the project owner confirmed that this is a valid behaviour.